gradio/components/base.py

```diff
@@ -133,7 +133,7 @@ def __init__(
         value: Any = None,
         label: str | None = None,
         info: str | None = None,
-        show_label: bool = True,
+        show_label: bool | None = None,
```

Comment:
Would be good to update the docstring to reflect that `show_label=None` falls back to the `container` param